manufacturing-ai-suite/industrial-edge-insights-time-series/configs/nginx/nginx.conf

```diff
@@ -23,10 +23,28 @@ http {
     # Host mapping is defined in the Docker Compose file
     server {
         listen 15443 ssl;
+
+        server_name localhost;
+
+        client_max_body_size 500M;
         # Nginx SSL cert paths
         ssl_certificate /opt/nginx/certs/cert.pem;
         ssl_certificate_key /opt/nginx/certs/key.pem;
 
+        # SSL security settings
+        ssl_protocols TLSv1.2 TLSv1.3;
+        ssl_ciphers "ECDHE+AESGCM:DHE+AESGCM:AES256+ECDHE:AES256+DHE";
```

Comment:
For line 34 and 35, can we try once having only TLSv1.3 and it's corresponding ssl_ciphers used in line 35 like "TLS_AES_256_GCM_SHA384".